docs/04-api-reference/front-commerce-remix/index.mdx

```diff
@@ -31,25 +33,56 @@ the `@front-commerce/remix` reference to your `remix.env.d.ts` file.
 /// <reference types="@front-commerce/remix" />
 ```
 
-### Improved loader data types
```

Comment:
I removed this as it resulted in reduced visibility for the `@front-commerce/remix/node` and  `@front-commerce/remix/react` api's this also now allows for subtitles in each of those sections.